packages/valory/skills/registration_abci/payloads.py

```diff
@@ -34,33 +34,9 @@ def __str__(self) -> str:
         return self.value
 
 
+@dataclass(frozen=True)
 class RegistrationPayload(BaseTxPayload):
     """Represent a transaction payload of type 'registration'."""
 
+    initialisation: str
```

Comment:
No longer optional. Optional fields should not be used in the current implementation, since the DictToProtobuf serializer cannot deal with `None`s

Reply:
We're removing DictToProtobuf though, so just add a TODO so we can fix this in the next wave